docs/release-notes.rst

```diff
@@ -1,3 +1,216 @@
+
+AS3 applications will support either static nodes or dynamic nodes (using the **Manage Endpoint Mappings** button) but not both. 
+
+------
+
+**Dynamic endpoints will not be discovered if any of the dynamic endpoint IPs already belong to the static nodes on the BIG-IP**
+
+Dynamic endpoints are the endpoints present in APIC Endpoint Group. The app automatically updates this in the BIG-IP application’s pool members provided the correct association is configured via the application. But if this dynamic endpoint list consists of an IP which has previously been added as a static node on the BIG-IP, none of the dynamic endpoints will get updated in the application. 
+
+**Workaround:** Ensure that the APIC endpoint subnet/IPs are different from the static endpoint IPs on the BIG-IP.
+
+------
+
+**Same Dynamic endpoint mappings on two separate partitions of a BIG-IP are not supported**
+
+For a single BIG-IP device, if two AS3 applications belonging to two different partitions are associated with the same APIC Endpoint Group (Tenant|Application|EPG), the dynamic discovery of nodes will not work for either of the AS3 applications. 
+
+**Workaround:** If you want to use the same endpoint mapping for two AS3 applications belonging to two different partitions, use the **shareNodes** option as described in https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/miscellaneous.html#using-sharenodes-to-reuse-nodes-across-tenants. 
+
+To enable shareNodes,
+
+1. Go to L4-L7 App Services --> Application --> Advanced.
+
+2. Create a new partition/application.
+
+3. Set dynamic endpoint mappings via **Manage Endpoint Mappings**, by selecting the Tenant|Application|EPG and port and click **Save**.
+
+4. Update the members section as below to add the shareNodes property:
+
+Example: "members": [
+            {
+                "addressDiscovery": "event",
+                
+                "servicePort": 80,
+                
+                "shareNodes": true
+            }
+         ]
+
+Another possible workaround is to remove the erroneous applications and recreate them with different mappings so that each AS3 application will have a separate set of nodes.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/187
+
+------
+
+**BIG-IP reboots OR BIG-IP services restart if more than 60 endpoints are dynamically discovered in an APIC EPG**
+
+If an APIC endpoint group has more than 60 endpoints attached, then the endpoint list will not get reflected on the BIG-IP, and users may experience service restarts on BIG-IP. 
+
+**Workaround:** For any dynamic endpoint mapping, ensure that the number of endpoints in the corresponding APIC endpoint group never exceeds 60.
+
+**AS3 Defect:** https://github.com/F5Networks/f5-appsvcs-extension/issues/188
+
+------
+
+**Node not removed from BIG-IP pool when node IP is a substring of some other node's IP**
```

Comment:
"A node is not removed from a BIG-IP pool when the node IP..."
or
"Nodes are not removed from a BIG-IP pool when the node IP..."